### ISSUE TITLE
Fist of the North Star: wrong cover and metadata

### ISSUE THREAD

**Issue by legolas119** [mrneo240/openmenu#25]
Hi
I added to GDEMU FIst of the North Star, the atomiswave port. The game works fine, but the cover and metadata are wrong, the game is associated to THE RUMBLE FISH:
https://i.postimg.cc/59v7Xpy9/IMG-3235.jpg

Is it possible to fix it, please?

And when i load it with GDMENU Card Manager, it detects it with a little typing error (FIRST instead of FIST): https://i.postimg.cc/vH0WT6rH/Fist.png

thanks

**Comment by mrneo240** [comment]
Name is pulled from the header of the ip.bin in the game image,  you can edit by clicking it.

Are you using the gdi versions or the cdi? I believe it should be handled correctly with the latest metadata DATs and the latest openMenu build.

I'll verify later tomorrow

**Comment by megavolt85** [comment]
use only original GDI

![Снимок экрана от 2021-12-28 23-48-05](https://user-images.githubusercontent.com/10085283/147605956-1fbc40e7-efdd-4740-bb2f-dd9a8b314861.png)

**Comment by legolas119** [comment]
yes, i'm using the .gdi image. I'm sorry but i haven't understood how to fix it.. please can you help me?

**Comment by legolas119** [comment]
hi, just to know if i have correctly understood: you said that you will try to fix this bug, is that correct? Many thanks!

**Comment by legolas119** [comment]
hi,
i'm sorry, do you have any news about the fix of the 2 little issues i explained in this topic? many thanks!

**Comment by legolas119** [comment]
hi, sorry, is this bug fixed? Or it will be fixed in the future? thanks!

**Comment by legolas119** [comment]
hi, is it possible to fix this bug? thank you very much
